.github/actions/annotate/index.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs');
+const https = require('https');
+
+// Function to strip versioning from package names
+function stripVersion(packageLine) {
+    return packageLine.split(/[<>=~!]/)[0].trim();
+}
+
+// Function to make an API request using the built-in https module
+function makeApiRequest(packageName) {
+    const url = `https://openteams-score.vercel.app/api/package/pypi/${packageName}`;
+
+    return new Promise((resolve, reject) => {
+        https.get(url, (res) => {
+            let data = '';
+
+            // Collect the data as it comes in
+            res.on('data', (chunk) => {
+                data += chunk;
+            });
+
+            // End of the response
+            res.on('end', () => {
+                if (res.statusCode === 200) {
+                    resolve(JSON.parse(data));
+                } else {
+                    reject(`Request failed with status code ${res.statusCode}`);
+                }
+            });
+        }).on('error', (e) => {
+            reject(`Error: ${e.message}`);
+        });
+    });
```

Comment:
can you replace this with the fetch builtin?
```suggestion
   const res = await fetch(url);
   return res.json()
```